geode-core/src/main/java/org/apache/geode/internal/offheap/ReferenceCountHelper.java

```diff
@@ -81,72 +85,72 @@ public static Object createReferenceCountOwner() {
    * of this method must also call unskipRefCountTracking after the allocation or free is done.
    */
   public static void skipRefCountTracking() {
-    getInstance().skipRefCountTracking();
+    delegate().skipRefCountTracking();
   }
 
   /**
    * Returns true if currently tracking reference counts.
    */
-  public static boolean isRefCountTracking() {
-    return getInstance().isRefCountTracking();
+  static boolean isRefCountTracking() {
```

Comment:
This method (and several others in this class) are unused. Would it be appropriate to remove them?

Reply:
be careful with this. You could check with Eric but I think some of these methods are used when trying to debug offheap leaks (i.e. orphans)

Reply:
I'm going to keep the unused methods in this class and restore the methods to public. We can clean this class up further next time.